[compat] test_system_views.py.TestSystemViewsRegistry.test_domain_sys_dir[restart_stab...: test_domain_sys_dir NoneType not callable

## Summary
test_domain_sys_dir NoneType not callable

## Environment
- Branch: main `345cf9df9923a4d214bfe902b2bd2e723b24bfa3`
- Compat chain: stable-26-1-1 `247d5be0d7b2fbdeb3f8ca920edc23cf35ffcc36` -> stable-26-2-1 `abac06dc12ea0a91479fb3aabbf7e467cb102f82` -> stable-26-3-1 `b1e96c302f42b329d2387d4b4ae451347aa9c47d`
- CI: [29327138619](https://github.com/ydb-platform/ydb/actions/runs/29327138619)
- Reports: [try1](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_1/ya-test.html), [try2](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_2/ya-test.html), [try3](https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_3/ya-test.html)

- **Full log:** https://storage.yandexcloud.net/ydb-gh-logs/ydb-platform/ydb/Regression-run_compatibility/29327138619/ya-main-x86-64/try_3/artifacts/logs/ydb/tests/compatibility/common/test-results/py3test/testing_out_stuff/test_system_views.py.TestSystemViewsRegistry.test_domain_sys_dir.restart_stable-26-2-1_to_stable-26-3-1.log
- Failed attempts: try_1, try_2, try_3

## Reproduction
```
-DYDB_COMPAT_INIT_REF=stable-26-2-1 -DYDB_COMPAT_INTER_REF=stable-26-3-1 -DYDB_COMPAT_TARGET_REF=prestable-26-4
```

## Tests (1)
```
ydb/tests/compatibility/common/test_system_views.py.TestSystemViewsRegistry.test_domain_sys_dir[restart_stable-26-2-1_to_stable-26-3-1]
```

## Error
```
ydb/tests/compatibility/common/test_system_views.py:68: in test_domain_sys_dir
    sysview_folder_content_before = self.collect_sysviews()
ydb/tests/compatibility/common/test_system_views.py:25: in collect_sysviews
    response = session.describe_table(f'/Root/.sys/{sysview.name}')
contrib/python/ydb/py3/ydb/table.py:1932: in describe_table
    return self._driver(
contrib/python/ydb/py3/ydb/tracing.py:77: in wrapper
    return f(self, *args, **kwargs)
contrib/python/ydb/py3/ydb/pool.py:541: in __call__
    res = connection(
contrib/python/ydb/py3/ydb/connection.py:557: in __call__
    return response if wrap_result is None else wrap_result(rpc_state, response, *wrap_args)
contrib/python/ydb/py3/ydb/_session_impl.py:20: in decorator
    return func(rpc_state, response_pb, session_state, *args, **kwargs)
contrib/python/ydb/py3/ydb/_session_impl.py:110: in wrap_describe_table_response
    return scheme._wrap_scheme_entry(
contrib/python/ydb/py3/ydb/scheme.py:423: in _wrap_scheme_entry
    return scheme_entry_cls(
contrib/python/ydb/py3/ydb/table.py:1530: in __init__
    self.columns = [Column(column.name, convert.type_to_native(column.type), column.family) for column in columns]
contrib/python/ydb/py3/ydb/convert.py:271: in type_to_native
    return _type_to_native_map.get(type_pb.WhichOneof("type"))(type_pb)
E   TypeError: 'NoneType' object is not callable```

--- a/ydb/tests/compatibility/common/test_system_views.py
+++ b/ydb/tests/compatibility/common/test_system_views.py
@@ -6,6 +6,8 @@ from ydb.tests.oss.ydb_sdk_import import ydb
 
 logger = logging.getLogger(__name__)
 
+pg_sysviews = {'pg_tables', 'tables', 'pg_class'}
+
 
 class TestSystemViewsRegistry(RestartToAnotherVersionFixture):
     @pytest.fixture(autouse=True, scope='function')
@@ -20,6 +22,9 @@ class TestSystemViewsRegistry(RestartToAnotherVersionFixture):
         with ydb.SessionPool(self.driver, size=1) as pool:
             with pool.checkout() as session:
                 for sysview in self.driver.scheme_client.list_directory('/Root/.sys').children:
+                    if sysview.name in pg_sysviews:
+                        continue
+
                     sysview_descr = dict()
 
                     response = session.describe_table(f'/Root/.sys/{sysview.name}')

